paimon-web-ui-new/mock/modules/metadata.js

```diff
@@ -15,10 +15,10 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License. */
 
-module.exports = {
-  '/catalog/getAllCatalogs': {
+module.exports = (mockUtil) => ({
+  '/catalog/getAllCatalogs': mockUtil({
     code: 200,
-    message: 'success',
+    msg: 'Successfully',
```

Comment:
```suggestion
    msg: 'Success',
```

Reply:
This is the same message as the backend result.